docs/model_developers/projects.rst

```diff
@@ -0,0 +1,17 @@
+Understanding Projects
```

Comment:
Does this add more than confusion?

Reply:
I'm not sure. It's a nicer overview than but some duplication of what's already in `model_developers/project_creation.rst`. I think it might be better to just add a mention of the requirements file there. But I have no real preference.

Reply:
This is fine for me. It's consistent with the structure we already have for modules.